pyproject.toml

```diff
@@ -46,3 +46,7 @@ force_grid_wrap = 0
 use_parentheses = true
 line_length = 100
 ensure_newline_before_comments = true
+
+[tool.semantic_release]
+version_variable = "setup.py:__version__"
+upload_to_repository = false
```

Comment:
Is this all the config that's needed? I'm surprised you don't need to setup to release from the main branch, do a git tag etc.

Reply:
Based on https://python-semantic-release.readthedocs.io/en/latest/, it really is :-D But your point with the branch is on a spot - it defaults to master, while we are using main.